packages/cli/src/commands/integration/list.ts

```diff
@@ -37,11 +39,15 @@ export async function list(client: Client) {
   // Note: the `--integration` flag is tracked later, after validating
   // whether the value is a known integration name or not.
 
-  if (parsedArguments.args.length > 2) {
-    output.error(
-      'Cannot specify more than one project at a time. Use `--all` to show all resources.'
-    );
-    return 1;
+  const validationResult = validateLsArgs(
+    'integration list',
```

Comment:
Should we just use `integration list [project]` here, and then we can omit the last arg and rely on the default?